resolvers/README.md

```diff
@@ -212,7 +241,7 @@ Here is most of the [Node resolver] at the time of this writing. It is just a wr
 var resolve = require('resolve/sync')
 var isCoreModule = require('is-core-module')
 
-exports.resolve = function (source, file, config) {
+exports.resolve = function (source, file, config, _, options) {
```

Comment:
`config` and `options` are easy confusions.

Reply:
What's your favor?